lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Transactions.hs

```diff
@@ -656,6 +660,51 @@ spec = describe "SHELLEY_TRANSACTIONS" $ do
                 (#balance . #available)
                 (`shouldBe` Quantity (faucetAmt - feeEstMax - amt)) ra2
 
+    it "TRANS_CREATE_10 - Multi-asset balance" $ \ctx -> runResourceT $ do
+        w <- fixtureMultiAssetWallet ctx
+        r <- request @ApiWallet ctx (Link.getWallet @'Shelley w) Default Empty
+        verify r
+            [ expectField (#assets . #available) (`shouldNotBe` mempty)
+            , expectField (#assets . #total) (`shouldNotBe` mempty)
+            ]
+
+    it "TRANS_CREATE_10 - Multi-asset transaction" $ \ctx -> runResourceT $ do
+        liftIO $ pendingWith "fee too small"
+
+        wSrc <- fixtureMultiAssetWallet ctx
+        wDest <- emptyWallet ctx
+        ra <- request @ApiWallet ctx (Link.getWallet @'Shelley wSrc) Default Empty
+        let (_, Right wal) = ra
+
+        -- pick out an asset to send
+        let assetsSrc = wal ^. #assets . #total . #getApiT
+        assetsSrc `shouldNotBe` mempty
+        let TokenBundle.AssetId pid an = Set.elemAt 0 (TokenMap.getAssets assetsSrc)
+        let val = [(toText pid, toText an, minUTxOValue)]
+
+        payload <- liftIO $ mkTxPayloadMA ctx wDest minUTxOValue val fixturePassphrase
```

Comment:
Seeing some strange things removing the `pendingWith` after rebasing (not sure it is the same as what @rvl saw):

### As is here, specifying `minUTxOValue` ada

<details>
<summary>Wallet error: InsufficientMinCoinValueError: 1 ada</summary>

```
"{\\\"code\\\":\\\"utxo_too_small\\\",\\\"message\\\":\\\"Some outputs have ada values that are too small. There's a minimum ada value specified by the protocol that each output must satisfy. I'll handle that minimum value myself when you do not explicitly specify an ada value for an output. Otherwise, you must specify enough ada. Here are the problematic outputs: InsufficientMinCoinValueError {outputWithInsufficientAda = TxOut {address = Address {unAddress = \\\\\\\"\\\\\\\\SOH\\\\\\\\238\\\\\\\\229Q\\\\\\\\178\\\\\\\\149\\\\\\\\244\\\\\\\\165`\\\\\\\\169Z\\\\\\\\250\\\\\\\\208AC@\\\\\\\\vx\\\\\\\\165\\\\\\\\242\\\\\\\\f\\\\\\\\254\\\\\\\\196\\\\\\\\232\\\\\\\\160[\\\\\\\\225kQ\\\\\\\\197\\\\\\\\196\\\\\\\\ETX\\\\\\\\148\\\\\\\\185/\\\\\\\\186\\\\\\\\&7S\\\\\\\\179\\\\\\\\153Ao\\\\\\\\EM\\\\\\\\221\\\\\\\\SO\\\\\\\\191\\\\\\\\239\\\\\\\\251\\\\\\\\173\\\\\\\\131\\\\\\\\176\\\\\\\\236\\\\\\\\DLE\\\\\\\\234+q\\\\\\\\a\\\\\\\"}, tokens = TokenBundle {coin = Coin 1000000, tokens = TokenMap (fromList [(UnsafeTokenPolicyId (Hash \\\\\\\"K\\\\\\\\254z\\\\\\\\202\\\\\\\\225\\\\\\\\189%\\\\\\\\153d\\\\\\\\153b\\\\\\\\177F\\\\\\\\161\\\\\\\\228}.\\\\\\\\DC4\\\\\\\\147\\\\\\\\&8\\\\\\\\t\\\\\\\\179g\\\\\\\\232\\\\\\\\EOT\\\\\\\\198\\\\\\\\US\\\\\\\\134\\\\\\\"),NonEmptyMap {least = (UnsafeTokenName \\\\\\\"\\\\\\\",TokenQuantity 1000000), rest = fromList []})])}}, expectedMinCoinValue = Coin 1407406} :| []\\\"}\", responseCookieJar = CJ {expose = []}, responseClose' = ResponseClose}")
```
</details>

### When Specifying `0` ada

<details>
<summary> Ledger error: OutputTooSmallUTxO: 1 ada </summary>
from the following response: Left (DecodeFailure "{\"code\":\"created_invalid_transaction\",\"message\":\"That's embarrassing. It looks like I've created an invalid transaction that could not be parsed by the node. Here's an error message that may help with debugging: HardForkApplyTxErrFromEra S (S (S (Z (WrapApplyTxErr {unwrapApplyTxErr = ApplyTxError [LedgerFailure (UtxowFailure (UtxoFailure (OutputTooSmallUTxO [(Addr Mainnet (KeyHashObj (KeyHash \\\"e4674505240c98422ad66a98ecdaca42770fa7c66a2bf1f81048c5ad\\\")) (StakeRefBase (KeyHashObj (KeyHash \\\"211fa28b02df352f6a5e93f9c920b967e5359180cb36b499093647f3\\\"))),Value 1407406 (fromList [(PolicyID {policyID = ScriptHash \\\"4bfe7acae1bd2599649962b146a1e47d2e14933809b367e804c61f86\\\"},fromList [(\\\"\\\",1000000)])]))])))]}))))\"}")
</details>

Reply:
That second error is quite bad. Sounds like something isn't working properly. When specyfing 0 Ada, the server should assign the minimum ada value needed for that UtxO. And, the first error suggests that the server is perfectly able to compute that minimum value because it suggests it in the error message. So, something is off..

Reply:
> As is here, specifying minUTxOValue ada

As far as I understand it is expected, since `minUTxOValue` may be different for MA. (Correct me if I'm wrong)

> When Specifying 0 ada

Interesting, because I don't see it on `mary_qa`. When specifying 0 ADA on tx with MA the transaction is given `minUTxOValue` correctly and tx is succesful.

Reply:
The first case is indeed expected, and the error returned by the server looks legit (1.4 Ada of expected min coin value). The second error is puzzling. From the message however, we can't really see if the error is on the change output or on the requested output.

Reply:
The "expected" min ada behaviour is a little odd, but I have amended the test cases accordingly. Perhaps the HTTP status code should be 400 Bad Request instead of 403 Forbidden.

I can't reproduce the second error in the integration tests.

Reply:
> The "expected" min ada behaviour is a little odd

What do you mean? Do you find the server behavior odd, or the ledger requirement odd? 

> Perhaps the HTTP status code should be 400 Bad Request instead of 403 Forbidden.

That's a semantic debate, yet I do think of `400 Bad Request` as requests which are malformed (improper JSON, invalid types, missing required keys etc..). Though, if the request is syntactically valid, but only present an invalid combination of inputs, then I rather go for 403.

Reply:
> I can't reproduce the second error in the integration tests.

You mean that, passing a value of 0 Ada has the intended behavior (that is, min ada value is automatically assigned, and the transaction go through) ?

Reply:
Oops, I think the `UtxoFailure (OutputTooSmallUTxO` I saw was since I was still on 1.24.2 locally.